src/main/java/org/lanternpowered/server/game/registry/type/block/BlockRegistryModule.java

```diff
@@ -1090,6 +1238,71 @@ public void registerDefaults() {
                                 .add(PropertyProviderCollections.UNBREAKABLE))
                         .translation("tile.barrier.name")
                         .build("minecraft", "barrier"));
+        //////////////////////
+        ///   Prismarine   ///
+        //////////////////////
+        register(168, simpleBuilder()
+                        .trait(LanternEnumTraits.PRISMARINE_TYPE)
+                        .defaultState(state -> state.withTrait(LanternEnumTraits.PRISMARINE_TYPE, LanternPrismarineType.ROUGH).get())
+                        .itemType(builder -> builder.keysProvider(valueContainer -> valueContainer
+                            .registerKey(Keys.PRISMARINE_TYPE, LanternPrismarineType.ROUGH)))
```

Comment:
`.keysProvider` starting on the next line to be consistent with the other registrations